src/gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,231 @@
+            y=self.probability["y"],
+            title="gaussian_pdf",
+            fig_processing='save'
+        )
+        distribution_fig_path = Visualizer.visualize(
+            x=self.distribution["x"],
+            y=self.distribution["y"],
+            title="gaussian_cdf",
+            fig_processing='save'
+        )
+        return probability_fig_path, distribution_fig_path
+
+
+class BetaProbabilisticSetting(ProbabilisticDistribution):
+    """Setting for beta distribution
+    """
+
+    def __init__(self) -> None:
+        """Set data points and compute probability for beta distribution
+        """
+        probability_x = np.linspace(0, 1, 100)
+        a = 0.5
+        b = 0.5
+        probability_y = []
+        for i in range(len(probability_x)):
+            probability_y.append(float(Distribution.beta_pdf(probability_x[i], a, b)))
+        probability_y = np.array(probability_y)
+
+        self.probability = {"x": probability_x, "y": probability_y}
+
+        distribution_x = probability_x
+        distribution_y = stats.beta.cdf(distribution_x, a, b)  # beta cumulative distribution function (beta cdf)
+        self.distribution = {"x": distribution_x, "y": distribution_y}
+
+    def show_figure(self) -> None:
+        """Show beta distribution figure
+        """
+        probability_fig_path = Visualizer.visualize(
+            x=self.probability["x"],
+            y=self.probability["y"],
+            title="beta_pdf",
+            fig_processing='show'
+        )
+        distribution_fig_path = Visualizer.visualize(
+            x=self.distribution["x"],
+            y=self.distribution["y"],
+            title="beta_cdf",
+            fig_processing='show'
+        )
+        return probability_fig_path, distribution_fig_path
+
+    def save_figure(self) -> None:
+        """Save beta distribution figure
+        """
+        probability_fig_path = Visualizer.visualize(
+            x=self.probability["x"],
+            y=self.probability["y"],
+            title="beta_pdf",
+            fig_processing='save'
+        )
+        distribution_fig_path = Visualizer.visualize(
+            x=self.distribution["x"],
+            y=self.distribution["y"],
+            title="beta_cdf",
+            fig_processing='save'
+        )
+        return probability_fig_path, distribution_fig_path
+
+
+class Distribution:
+
+    @staticmethod
+    def gauss_np(t: float, mu: float = 0, sigma: float = 1) -> float:
+        """Compute gaussian probability density function.
+
+        Args:
+            t: data point
+            mu: mean of gaussian probability density function
+            sigma: standard deviation of gaussian probability density function
+        Returns:
+            gaussian probability density function
+        """
+        return np.exp(- (t - mu) ** 2 / 2*sigma ** 2) / np.sqrt(2*np.pi*(sigma ** 2))
+
+    @staticmethod
+    def gauss_sp(t, mu: float = 0, sigma: float = 1):
```

Comment:
write return type

Reply:
Return type is sympy.Symbol, but when I write return type in this line, error occurred. So I wrote its in docstring.

Reply:
Then you should show the error descriptions.
I think you can solve it like:

```py
def gauss_sp(t, mu: float = 0, sigma: float = 1) -> Symbol:
```

because of your import statement is like:
```py
from sympy import Symbol
```